docs/BEMAN_STANDARD.md

```diff
@@ -200,7 +200,7 @@ endif()
 
 # ...
 if(<short_name>_BUILD_TESTING)
-  add_subdirectory(/path/to/tests)
+  add_subdirectory(tests)
```

Comment:
If DIRECTORY.TESTS is a requirement, then in the root CMakeLists we'll always have the same instruction: ` add_subdirectory(tests)`.